README.md

```diff
@@ -1,17 +1,22 @@
-# dbus-shelly-1pm-pvinverter
-Integrate Shelly 1PM into Victron Energies Venus OS
+# dbus-shelly-em-smartmeter
```

Comment:
we should think about a better name, since this is then usable for both devices 1pm and em. Do you have a suggestion?